templates/chats.html

```diff
@@ -5,12 +5,23 @@
       <div class="col-md-6">
         <h2>⌘ StartIT tērzētava</h2>
       </div>
-      <div class="col-md-4">
+      <div class="col-md-2">
         <p id="versija">JS versija: d(-_o)b </p>
       </div>
       <div class="col-md-2">
         <button id="skanjasPoga" value="up"><i class="fa fa-volume-up" id="ikona"></i></button>
       </div>
+      <div class="col-md-2">
+        <div class="dizaini">
+          <button class="dizainpoga">Dizains</button>
+            <div class = "dizaini-saturs">
+              <button type="button" onclick="krasasMaina('tumss');">Tumšs</button>
+              <button type="button" onclick="krasasMaina('bg-white');">Gaišs</button>
```

Comment:
Kāpēc pārējiem nav *bg-tumss*, *bg-zals*?

Reply:
Aizrādījums vietā. bg_white jau bija kodā, tam nosaukumu mainīt neuzdrošinājos. Pielabošu savus veidotos nosaukumus.